src/components/CommentForm.js

```diff
@@ -87,8 +96,12 @@ export default class CommentForm extends React.Component {
 
     const setText = event => updateStore(event.target.value)
     const placeholder = this.props.placeholder || 'Add a comment...'
+
+    const keyUp = debounce(stoppedTyping, STOPPED_TYPING_WAITTIME)
+    const startThrottled = throttle(startedTyping, MAX_TYPING_INTERVAL, { trailing: false })
     const keyDown = e => {
       this.setState({enabled: this.refs.editor.getContent().length > 0})
+      startThrottled()
       onCmdOrCtrlEnter(e => {
         e.preventDefault()
         this.submit()
```

Comment:
just realized that if the form is submitting it should/could call `stoppedTyping()`